virttest/utils_windows/virtio_win.py

```diff
@@ -132,26 +132,8 @@ def _get_netkvmco_path(session):
         raise exceptions.TestError(err)
 
     middle_path = "%s\\%s" % (guest_name, guest_arch)
-    find_cmd = 'dir /b /s %s\\netkvmco.dll | findstr "\\%s\\\\"'
-    find_cmd %= (viowin_ltr,  middle_path)
+    find_cmd = 'dir /b /s "%s\\" | findstr "%s" | findstr "%s"'
+    find_cmd %= (viowin_ltr,  middle_path, "netkvmco.exe")
```

Comment:
@heywji Would you please clarify why we need to modify the pattern? Will simply replace dll with exe introduce any regression?

Reply:
> @heywji Would you please clarify why we need to modify the pattern? Will simply replace dll with exe introduce any regression?

The fact that they all work(You can see [this PR](https://github.com/autotest/tp-qemu/pull/3858/files#diff-4a6a5e295efa78ddc2ac7d3de80f35ca6a0ca8474cc72cd873836c21aed9594bR175)) does not INTRODUCE any regression. I think the previous pattern was too inconvenient to understand. It has been modified, for example before it was `dir /b /s E:\netkvmco.exe | findstr "\2k22\amd64\\\"` now it is `dir /b /s E:\ \ | findstr "\2k22\amd64\\\ | findstr "netkvmco.exe"`

The `/b` parameter is used to output the results in plain text format without displaying detailed information about files and folders, while `/s` parameter is used to search for files in the specified directory and its subdirectories.